packages/cli/test/models/ProjectFile.test.js

```diff
@@ -24,22 +22,22 @@ contract('ProjectFile', function() {
     });
 
     it('loads existing file', function() {
-      const file = new ProjectFile(
-        'test/mocks/packages/package-with-contracts.zos.json',
-      );
+      const file = new ProjectFile('test/mocks/packages/package-with-contracts.zos.json');
       file.data.manifestVersion.should.eq(MANIFEST_VERSION);
       file.name.should.eq('Herbs');
       file.version.should.eq('1.1.0');
       file.contract('Impl').should.eq('ImplV1');
     });
 
     it('fails to load unsupported manifestVersion', function() {
-      expect(
-        () =>
-          new ProjectFile(
-            'test/mocks/packages/package-unsupported-manifest-version.zos.json',
-          ),
-      ).to.throw(/Unrecognized manifest version identifier 3/);
+      expect(() => new ProjectFile('test/mocks/packages/package-unsupported-manifest-version.zos.json')).to.throw(
+        /Unrecognized manifest version identifier 3/,
+      );
     });
   });
+
+  it('supports deprecated manifest version', function() {
+    const file = new ProjectFile('test/mocks/packages/package-deprecated-manifest-version.zos.json');
+    file.data.zosversion.should.eq('2.2');
```

Comment:
```suggestion
    file.manifestVersion.should.eq('2.2');
```

Shouldn't we be checking that the getter handles both formats?

Reply:
Actually, we'd need to add a getter to be able to do this (the network file has one, not the project file), so feel free to disregard this suggestion